### DISCUSSION TITLE
Out of Memory

### DISCUSSION THREAD

**Discussion by oguzc** [community/community#54857] [Codespaces]
### Select Topic Area

Bug

### Body

I got error Command failed with exit code 137 while running storybook for my app. it seems it's because of memory limitation which I understand. I can try to adjust the limit but right now my account is blocked almost completely, it cannot load anything.

**Comment by matinrasooli** [comment]
you can change list with set or dict or tuple....

**Comment by oguzc** [comment]
I'm not sure if this is related to my issue. I've just ran a react app on codespaces which broke my github account :)
